src/DatadogLoggingService.js

```diff
@@ -60,6 +60,8 @@ class DatadogLoggingService extends NewRelicLoggingService {
       sessionSampleRate: parseInt(process.env.DATADOG_SESSION_SAMPLE_RATE || 0, 10),
       sessionReplaySampleRate: parseInt(process.env.DATADOG_SESSION_REPLAY_SAMPLE_RATE || 0, 10),
       trackUserInteractions: true,
+      trackSessionAcrossSubdomains: true,
+      usePartitionedCrossSiteSessionCookie: true,
```

Comment:
[curious] Is this getting enabling largely due to how `@openedx/frontend-plugin-framework` supports consuming  private plugins as an iframe? Are there other use cases when a Datadog-enabled MFE might be loaded via an iframe?

Reply:
yeah, to the best of my knowledge we aren't currently even doing that, but my thought was that since FPF allows it,  we should probably enable data dog in an iframe without someone having to solve this problem all over again in the future. But I could also solve that with setting it to false, and adding a comment, I suppose. What you think?